distribution/docs/src/main/jdocs/content/_security-framework/realms-contents.adoc

```diff
@@ -0,0 +1,107 @@
+:title: Realms
+:type: subSecurityFramework
+:status: published
+:parent: Subject
+:order: 01
+:summary: Shiro realms.
+
+${branding} uses Realms for Authentication and Authorization.
+
+===== Authenticating Realms
+
+`org.apache.shiro.realm.AuthenticatingRealm`
+
+Authenticating Realms are used to authenticate an incoming authentication token and create a Subject on successful authentication.
+
+.Implementations of Authenticating Realms in ${branding}:
+[cols="1m,2" options="header"]
+|===
+
+|Classname
+|Description
+
+|${ddf-branding-lowercase}.security.realm.sts.StsRealm
+|This realm delegates authentication to the Secure Token Service (STS). It creates a `RequestSecurityToken` message from the incoming Authentication Token and converts a successful STS response into a Subject.
+
+|===
+
+===== Authorizing Realms
+
+`org.apache.shiro.realm.AuthorizingRealm`
+
+Authorizing Realms are used to perform authorization on the current Subject.
+These are used when performing both Service AuthZ and Filtering.
```

Comment:
❓ Why is `Filtering` capitalized? It is used in a special way or refer to a particular entity?

Reply:
removed the capital.